### DISCUSSION TITLE
Additional points for high horizons - Parallels of Declination

### DISCUSSION THREAD

**Discussion by JDAstroPhoto** [mworion/MountWizzard4#275] [Feature Request] [answered]
In my backyard I have variable horizons from 45 degrees to 10 degrees.  I like using parallels of Declination.  It is such a clean plot in the polar view.   For max points for my backyard, I am limited to 65 on Parallels of Declination.  On my dark sky spot, where I limit horizon to 25 degrees, of course it is much higher and I typically pick something around 85 points.  Plenty of points with those three buttons.
For my backyard:
I know I can always keep existing points and add additional points, but it tends to get a little messy.  It would be nice is we could give the number of points to Parallels of Declination and have it put them in, regardless of horizon.  I would like to do 85 in my backyard as well.
Thanks for your effort, it is appreciated.

**Comment by mworion** [comment, marked as answer]
Currently the number are a result out of a preset of points I did. This makes the arrangement in your eyes that clean. There is actually no algorithm, which generates them automatically based on desired number. As an intermediate solution, please enable showing the greater circles that you are able to set manual points more accurate.
I put it an the wishlist for the future.
Michel

**Comment by mworion** [comment]
For the next version 3.0 you could give an exact number of points in golden spiral which respects all your settings of removing points or any other constraints.
Hope this helps.
Michel

**Reply by JDAstroPhoto** [reply]
Thank-you very much, your work is appreciated.
